templates/index.html

```diff
@@ -689,7 +689,7 @@ <h4>Participate</h4>
 							<a href="https://github.com/juju/juju" class="p-link--external">Juju on GitHub</a>
 						</li>
 						<li class="p-list__item">
-							<a href="https://discourse.juju.is/">Juju on Discourse</a>
+							<a href="https://discourse.charmhub.io/">Juju on Discourse</a>
```

Comment:
do you think we should put the category link here?
https://discourse.charmhub.io/c/juju

Reply:
Sure